testcases/multizone_office_simple_hydronic/doc/index.html

```diff
@@ -0,0 +1,1206 @@
+<ul>
+<li>North zone:
+<ul>
+<li>30 W/m<sup>2</sup> for heating</li>
+<li>40 W/m<sup>2</sup> for cooling</li>
+</ul>
+</li>
+<li>South zone:
+<ul>
+<li>25 W/m<sup>2</sup> for heating</li>
+<li>45 W/m<sup>2</sup> for cooling</li>
+</ul>
+</li>
+</ul>
+As such, the nominal powers of the emission system are 75 and 100 kW for heating and cooling respectively for the north zone
+and 62.5 and 112.5 kW for heating and cooling respectively for the south zone.
+The mass flow rates of the emission loops are determined using the relation:
+</p>
+<p align=\"center\">
+Q<sub>design</sub> = m<sub>flow</sub> c<sub>p</sub> ΔT
+</p>
+<p>
+where Q<sub>nominal</sub> is the nominal heat flow, m<sub>flow</sub> is the nominal mass flow rate,
+c<sub>p</sub> is the specific heat capacity of the media and ΔT is the nominal temperature difference.
+This nominal temperature difference is a design parameter and it is assumed to be 20 K and
+5 K for the water stream for heating and cooling respectively,
+and 10 K for the air stream (both for the heating and cooling coil) in the ventiloconvector.
+The heat exchange coefficients (UA) of the heat exchangers in the ventiloconvectors are computed by the model
+from these nominal values using the <a href=https://en.wikipedia.org/wiki/NTU_method>NTU method</a>.
+</p>
+
+<p>
+The ventilation system is composed by two individual air handling units (AHUs) per zone.
+These AHUs are equipped with a double mechanical flux system (i.e., a supply and a return fan),
+a heat recovery system and a heating and cooling coils.
+No humidifier/dehumidifer system is installed.
+No variable air volume (VAV) boxes are equipped in the ventilation ducts.
+However, the fans of the AHU can modulate their pressure head to achieve the required ventilation rate.
+The nominal supply and return air flow rates are computed based on the volume of the zones to be conditioned,
+and they are 3 kg/s and 2.5 kg/s of air respectively.
+Based on this nominal air flows, the nominal powers and nominal mass flow rates of the heating and cooling coils
+are calculated using the same relations as for the emission system.
+The nominal temperature differences in this case are 20 K and
+5 K for the water stream for heating and cooling respectively,
+and 30 K and 15 K for the air stream for heating and cooling respectively.
+As a result, for each individual AHU the nominal heating power is 90 kW
+whereas the nominal cooling power is 45 kW.
+The heat recovery exchanger is assumed to have a constant effectiveness of 84%.
+</p>
+
+<p>
+The production system is composed by a heat pump for heating and a chiller for cooling.
+The nominal powers of the production system are determined by the sum of the nominal powers
+of each emission and ventilation coil loop, which are 75+62.5+90+90=317.5 kW and 100+112.5+45+45=302.5 kW
+for heating and cooling respectively.
+The nominal mass flow rates are calculated from the resulting heating and cooling nominal powers.
```

Comment:
Using the same equation and dTs as the emission calculations?

Reply:
For production all the mass flow rates are summed up. However, it is good that you bring this up because I detected a flaw:

- The nominal deltaT used to compute the mass flow rates in emission is set to 5 K both in heating and cooling (heat pump system)
- For the AHU, (see Components.AirHandlingUnit), the nominal deltaT used to compute the nominal power of the coil AND the ventilation mass flow rate is 20 K for heating and 5 K for cooling. This should be changed to 5 K for heating (line 38)
- For the FCUs, the nominal deltaT used to compute the nominal power of the FCUs is set to 20K in heating and 5 K for cooling. This should be set to 5K for heating (line 67 in BuildingEmulators.Templates.Heating.FanCoilUnits)

Reply:
@dhblum 

Changed in my local branch. Also, changed the schematic to include the drawing of a HP

To be revised: Nominal powers of the different pumps (run a simulation and check their power) and update documentation accordingly, in this section:

"""
The pumps of the distribution and production system follow the [generic](https://github.com/open-ideas/IDEAS/blob/master/IDEAS/Fluid/Movers/Data/Generic.mo) curve of the model and have the following nominal pressure heads and powers:

    Production pumps:
        Heating: 6 m head and 470 W
        Cooling: 6 m head and 1.8 kW
    Distribution pumps:
        Emission:
            Heating north zone: 12 m head and 220 W
            Cooling north zone: 12 m head and 1.2 kW
            Heating south zone: 12 m head and 185 W
            Cooling south zone: 12 m head and 1.3 kW
        Ventilation:
            Heating north zone: 9 m head and 200 W
            Cooling north zone: 9 m head and 400 W
            Heating south zone: 9 m head and 200 W
            Cooling south zone: 9 m head and 400 W
"""

Reply:
Thanks @icupeiro.  Can you make a PR from your fork to IBPSA:issue465_multizone_office_simple_hydronic with the changes?

Reply:
Ok I recompiled and simulated the model and updated the pump powers in https://github.com/ibpsa/project1-boptest/pull/619/commits/79b3c85865ebc5286f7fc793175352028d334cb6.